src/utils/extractReactTokens.ts

```
@@ -0,0 +1,119 @@
+    // but not the main component index file (e.g., c_accordion.js)
+    return tokenPrefixes.some((prefix) => {
+      if (file === `${prefix}.js`) {
+        // This is the main component file, skip it
+        return false
+      }
+      return file.startsWith(prefix)
+    })
+  })
+
+  // Import and extract objects from each matching file
+  const tokenObjects: Array<{ name: string; value: string; var: string }> = []
```

ESLint

[error] Array type using 'Array<T>' is forbidden. Use 'T[]' instead.

(`@typescript-eslint/array-type`)

---

[error] Unexpected console statement.

(no-console)